src/UDS.Net.API/Entities/D1b.cs

```diff
@@ -0,0 +1,316 @@
+        public int? CSFOTH { get; set; }
+
+        [Comment("4d1. CSF-based biomarkers - Consistent with other etiology (specify)")]
+        [MaxLength(60)]
+        public string? CSFOTHX { get; set; }
+
+        [Comment("5. Imaging - Was imaging used for assessing etiological diagnosis?")]
+        public int? IMAGINGDX { get; set; }
+
+        [Comment("6a. Tracer-based PET - Were tracer-based PET measures used in assessing an etiological diagnosis?")]
+        public int? PETDX { get; set; }
+
+        [Comment("6a1. Elevated amyloid")]
+        public int? AMYLPET { get; set; }
+
+        [Comment("6a2. Elevated tau pathology")]
+        public int? TAUPET { get; set; }
+
+        [Comment("6b. FDG PET - Was FDG PET data or information used to support an etiological diagnosis?")]
+        public int? FDGPETDX { get; set; }
+
+        [Comment("6b1. FDG PET - Consistent with AD")]
+        public int? FDGAD { get; set; }
+
+        [Comment("6b2. FDG PET - Consistent with FTLD")]
+        public int? FDGFTLD { get; set; }
+
+        [Comment("6b3. FDG PET - Consistent with LBD")]
+        public int? FDGLBD { get; set; }
+
+        [Comment("6b4. FDG PET - Consistent with other etiology")]
+        public int? FDGOTH { get; set; }
+
+        [Comment("6b4a. FDG PET - Consistent with other etiology (specify)")]
+        [MaxLength(60)]
+        public string? FDGOTHX { get; set; }
+
+        [Comment("6c. Dopamine Transporter (DAT) Scan - Was DAT Scan data or information used to support an etiological diagnosis?")]
+        public int? DATSCANDX { get; set; }
+
+        [Comment("6d. Other tracer-based imaging - Were other tracer-based imaging used to support an etiological diagnosis?")]
+        public int? TRACOTHDX { get; set; }
+
+        [Comment("6d1a. Other tracer-based imaging - Were other tracer-based imaging used to support an etiological diagnosis? (specify)")]
+        [MaxLength(60)]
+        public string? TRACOTHDXX { get; set; }
+
+        [Comment("6d1. Other tracer-based imaging - Consistent with AD")]
+        public int? TRACERAD { get; set; }
+
+        [Comment("6d2. Other tracer-based imaging - Consistent with FTLD")]
+        public int? TRACERFTLD { get; set; }
+
+        [Comment("6d3. Other tracer-based imaging - Consistent with LBD")]
+        public int? TRACERLBD { get; set; }
+
+        [Comment("6d4. Other tracer-based imaging - Consistent with other etiology")]
+        public int? TRACEROTH { get; set; }
+
+        [Comment("6d4a. Other tracer-based imaging - Consistent with other etiology (specify)")]
+        [MaxLength(60)]
+        public string? TRACEROTHX { get; set; }
+
+        [Comment("7a. Structural Imaging (i.e., MRI or CT) - Was structural imaging data or information used to support an etiological diagnosis?")]
+        public int? STRUCTDX { get; set; }
+
+        [Comment("7a1. Atrophy pattern conistent with AD")]
```

Comment:
Another spelling error in the documentation, so not your fault at all

conistent -> consistent